avogadro/core/slaterset.h

```diff
@@ -141,6 +140,7 @@ class AVOGADROCORE_EXPORT SlaterSet : public BasisSet
   MatrixX& normalizedMatrix() { return m_normalized; }
   MatrixX& densityMatrix() { return m_density; }
 
+  // Does nothing?
```

Comment:
In general, this type of basis set isn't used much. Like maybe 0.01% of the time. We've included it for backwards compatibility with a few programs.